core/storage/email/gae_models.py

```diff
@@ -127,7 +128,8 @@ def create(
         email_model_instance.put()
 
     def put(self):
-    	"""Creates the hash value of the instance of SentEmailModel."""
+        """Creates a hash value and saves this SentEmailModel 
```

Comment:
Drop "Creates a hash value and". That's really an implementation detail, meant to make the saved instance easily-findable. The point of this method (as far as its callers are concerned) is to save the instance to the datastore.